### ISSUE TITLE
Text - Onload Autogenerated Font Color

### ISSUE THREAD

**Issue by steckReed** [steckReed/npm-Full-Steck-Tools#13]

**Comment by steckReed** [comment]
https://github.com/steckReed/npm-Full-Steck-Tools/commit/b3119d8cd9bdbb955361444538df115430d3cba6

Method could be improved.
Currently using 35 for threshold, but is subject to change. (ln [227)](https://github.com/steckReed/npm-Full-Steck-Tools/commit/b3119d8cd9bdbb955361444538df115430d3cba6#r89319339)
Should be implemented in all mixins, but with overloaded params such as 'AA, AAA etc.' ratings, font-size, and font-weight
